cocotb/_build_libs.py

```diff
@@ -356,7 +381,7 @@ def get_ext():
         mti_path = os.path.join(modelsim_include_dir, "mti.h")
         if os.path.isfile(mti_path):
             fli_ext = Extension(
-                os.path.join("cocotb", "libs", "modelsim", "libfli"),
+                os.path.join("cocotb", "libs", "libfli_modelsim"),
```

Comment:
Given `fli` is a modelsim-only thing, is there any point having the suffix?

Reply:
Just consistency. Any is OK for me.